app/build.gradle

```diff
@@ -106,3 +129,5 @@ dependencies {
 
     implementation 'com.sylversky.fontreplacer:fontreplacer:1.0'
 }
+
+apply plugin: 'com.google.gms.google-services'
```

Comment:
can you please comment this line out for now, so the build won't crash if google-services.json file is missing?